### ISSUE TITLE
Implicit reference to ActiveWorkbook needs a quick-fix

### ISSUE THREAD

**Issue by retailcoder** [rubberduck-vba/Rubberduck#1927]
Currently, the only available quick-fix for `ImplicitActiveWorkbookReferenceInspection` is the `IgnoreOnceQuickFix`. Users should be able to pick from two more alternatives:

 - Qualify member call with `ActiveWorkbook`
 - Qualify member call with `ThisWorkbook`

**Comment by daFreeMan** [comment]
It would be nifty if the code could identify any `Workbook` variables active in the procedure (local or parameter), and dynamically add them as quick fix options.

Of course, I have **no** idea how much work that would take, but it would be a braggable feature!

**Comment by retailcoder** [comment]
@daFreeMan you mean add a `Workbook` parameter to the procedure, and update all call sites? :wink: 

Nah, there's just too many ways this could go wrong. We'd be making assumptions that would be wrong 99% of the time!

**Comment by daFreeMan** [comment]
No, I mean identify all existing `workbook` variables and present them as `Quick Fix` options in addition to `ActiveWorkbook` and `ThisWorkbook`. We don't want to encourage poor coding habits. Also, isn't there an inspection yielding some sort of warning for `This|ActiveWorkbook` usage indicating that it's quite potentially fragile?

**Comment by retailcoder** [comment]
There isn't. `ThisWorkbook` is completely unambiguous BTW, and explicit references to `ActiveWorkbook` can be completely legit.

**Comment by daFreeMan** [comment]
`ThisWorkbook` is completely unambiguous until you have a long running process and the user gets bored and clicks on another workbook.

**Comment by retailcoder** [comment]
@daFreeMan no. `ThisWorkbook` always refers to the workbook the code is in, regardless of the selection. You mean `ActiveWorkbook`, which changes at user's whim.

**Comment by daFreeMan** [comment]
Ah! See, by avoiding both whenever possible, I've not learned the finer distinctions.
